modules/test/playwright/pages/site-admin-web/SiteSettingsLocalizationPage.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+import {SiteSettingsPage} from './SiteSettingsPage';
+
+export class SiteSettingsLocalizationPage {
+	readonly page: Page;
+	readonly saveButton: Locator;
+	readonly siteSettingsPage: SiteSettingsPage;
+
+	constructor(page: Page) {
+		this.page = page;
+		this.saveButton = page.getByRole('button', {name: 'Save'});
+		this.siteSettingsPage = new SiteSettingsPage(page);
+	}
+
+	async goto() {
+		await this.siteSettingsPage.goToSiteSetting('Localization');
+	}
+
+	async saveConfiguration() {
+		this.saveButton.click();
+		await waitForSuccessAlert(this.page);
+	}
+
+	async selectCustomDefaultLanguageOption() {
+		await this.page
+			.getByLabel(
+				'Define a custom default language and additional available languages for this site.'
+			)
+			.click();
+	}
+
+	async selectDefaultLanguageOption() {
+		await this.page.getByLabel('Use the default language options.').click();
+	}
+
+	async setCustomDefaultLanguage(languageOption: string) {
+		const selector = this.page.locator(
+			`select[name="_com_liferay_site_admin_web_portlet_SiteSettingsPortlet_TypeSettingsProperties--languageId--"]`
```

Comment:
Did you try to use Playwright's **L**ocator here instead of **l**ocator?

Reply:
yes, but i couldn't find a way to make it work, i can try again just to make sure

Reply:
If you tried and could not find, it is ok. I always confirm if the author tried using Locators, if not, then they should try it. You can leave it this way.

Reply:
hmm i think i got wrong what you said, i'll change it and you tell me if it's right okay?